client/src/app/components/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="wrapper">
+    <div class="cover-picture">
+        
+        <button mat-icon-button class="close-dialog-icon" mat-dialog-close>
+            <mat-icon>close</mat-icon>
+        </button>
+        <button mat-button class="edit-cover-picture">
+            <mat-icon>add_a_photo</mat-icon>
+            <!-- Revisit this -->
+            <span class="edit-cover-pic-text">Edit Cover Photo</span>
+        </button>
+        
+        <div class="profile-picture">
+            <img src="https://ollienollie.files.wordpress.com/2009/10/untitled-181.jpg" alt="profile picture"
+            class="profile-pic" />
+            <button mat-icon-button class="edit-profile-pic">
+                <mat-icon>add_a_photo</mat-icon>
+            </button>
+        </div>
+    </div>
+
+
+    <form class="container">
```

Comment:
since the above is going to be called container, you can change this to edit-profile-form